mlrun/utils/model_monitoring.py

```diff
@@ -241,3 +241,10 @@ def validate_errors_and_metrics(endpoint: dict):
                 }
             }
         )
+    # Validate key `uid` instead of `endpoint_id`
+    # For backwards compatibility reasons, we replace the `endpoint_id` with `uid` which is the updated key name
+    if model_monitoring_constants.EventFieldType.ENDPOINT_ID in endpoint:
+        endpoint[model_monitoring_constants.EventFieldType.UID] = endpoint[
+            model_monitoring_constants.EventFieldType.ENDPOINT_ID
+        ]
+        endpoint.pop(model_monitoring_constants.EventFieldType.ENDPOINT_ID)
```

Comment:
Why not keep both? Conceptually, the caller may be an old client that still expected the `ENDPOINT_ID` field in the result. We support old clients as long as the server is new (not vice-versa).

Reply:
right, we'll keep both